examples/src/main/java/com/squareup/moshi/recipes/ReadJsonString.java

```diff
@@ -0,0 +1,64 @@
+package com.squareup.moshi.recipes;
+
+import com.squareup.moshi.FromJson;
+import com.squareup.moshi.JsonAdapter;
+import com.squareup.moshi.JsonQualifier;
+import com.squareup.moshi.JsonReader;
+import com.squareup.moshi.JsonWriter;
+import com.squareup.moshi.Moshi;
+import com.squareup.moshi.ToJson;
+import java.io.IOException;
+import java.lang.annotation.Retention;
+import okio.Buffer;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+final class ReadJsonString {
+  @Retention(RUNTIME)
+  @JsonQualifier
+  public @interface JsonString {
```

Comment:
nit: this is a weird example because it looks like it should be part of the framework

Reply:
if it was a name like @LocationMetadata that would be less of-the-framework